src/esp/sim/Simulator.cpp

```diff
@@ -195,11 +195,6 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
     flextGLInit(Magnum::GL::Context::current());
 #endif
     renderer_->acquireGlContext();
-  } else {
-    CORRADE_ASSERT(
-        !Magnum::GL::Context::hasCurrent(),
-        "Simulator::reconfigure() : Unexpected existing context when "
-        "createRenderer==false", );
```

Comment:
I removed this assert as it is obsolete.
In this case, a context exists (GLFW) but the simulators are headless.